frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.java

```diff
@@ -946,7 +953,11 @@ private boolean restoreFromBundle(@Nullable Bundle savedInstanceState, @Nullable
 
             //Successfully restored state
             return true;
-        } catch (JSONException e) {
+        } catch (Throwable ex) {
+            mTagCount = 0;
+            mCurrentFrag = null;
+            mFragmentStacks.clear();
```

Comment:
Also added additional clean up because in some cases a "half" load happened and we got stuck in some incomplete state